crates/evm/src/fuzz/strategies/state.rs

```diff
@@ -260,7 +260,7 @@ pub fn collect_created_contracts(
 
     for (address, account) in state_changeset {
         if !setup_contracts.contains_key(&b160_to_h160(*address)) {
-            if let (true, Some(code)) = (&account.is_touched(), &account.info.code) {
```

Comment:
This is the only part that really touches the fuzz strategies. It leads me to believe—maybe account recording is not being done correctly in some edge cases?

Reply:
This "should" be fine, `is_touched` have the same functionality, and if it was wrong, bug would probably be a lot more serious and would pop on mainnet sync.